[2022-08-20 03:34:24,717] ERROR [django.request.log_response:241] Internal Server Error: /appointments/
[2022-08-20 03:43:14,159] ERROR [shared.exception_handler.__handle_http404:111] No Appointment matches the given query.
[2022-08-20 03:43:14,162] WARNING [django.request.log_response:241] Not Found: /appointments/100/
[2022-08-20 03:43:19,665] WARNING [django.request.log_response:241] Not Found: /appointments/-123/
[2022-08-20 04:58:18,149] WARNING [django.request.log_response:241] Bad Request: /user/me/appointments/
[2022-08-20 04:58:33,264] WARNING [django.request.log_response:241] Bad Request: /user/me/appointments/
[2022-08-20 04:58:37,613] WARNING [django.request.log_response:241] Bad Request: /user/me/appointments/
